frontend/app/controllers/Staff.scala

```diff
@@ -0,0 +1,16 @@
+package controllers
+
+import play.api.mvc.Controller
+import services.GuardianLiveEventService
+
+import scala.concurrent.Future
+
+trait Staff extends Controller {
+  val guLiveEvents = GuardianLiveEventService
+
+  def eventOverview = PermanentStaffNonMemberAction.async { implicit request =>
+     Future.successful(Ok(views.html.staff.eventOverview(guLiveEvents.events, guLiveEvents.eventsDraft)))
```

Comment:
You don't need `Future.successful` here, just remove the `.async` on the `Action`